src/components/Avatar.tsx

```diff
@@ -100,7 +100,7 @@ function Avatar({
     let iconColors;
     if (isWorkspace) {
         iconColors = StyleUtils.getDefaultWorkspaceAvatarColor(avatarID?.toString() ?? '');
-    } else if (useFallBackAvatar) {
+    } else if (useFallBackAvatar && !fallbackIcon) {
```

Comment:
Let's add a comment here like
`Assign the icon fill color only for the default avatar and not when the fallback icon is available`